### ISSUE TITLE
\cite{...} completion sometimes with empty bibitem data

### ISSUE THREAD

**Issue by FMuro** [James-Yu/LaTeX-Workshop#2111]
### Disable all the other extensions except for LaTeX Workshop, and check that you still see this issue. [Required]

You still see this issue?: Yes

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes

### If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: Yes

## Describe the bug [Required]

Using a bibtex file, after opening a document and triggering a \cite{...} completion, the bibitem information is only showed the first time, the second a subsequent times it appears empty. A small change in the .bib file, like adding a space, recovers the correct behavior, but again, only once.

### To Reproduce

1. Open a LaTeX document using a bibtex file.
2. Add two \cite{...}
3. The first time, you'll have extra information on the right of completion suggestions, but it appears empty the second time. Slight modifications of the .bib files are like if you start over.

### Expected behavior

bibitem information should never appear empty on the right when completing \cite{...}

## Logs [Required]

Please paste the whole log messages here, not parts of ones. It is very important to identify problems.

### LaTeX Workshop Output [Required]

```
To access the log, click the 'TeX' icon on the Activity Bar on the left side, select 'View Log Messages', then select 'View LaTeX Workshop extension log'.

[Paste the log here]
```

[02:18:05] Initializing LaTeX Workshop.
[02:18:05] Creating file watcher for .pdf files.
[02:18:06] Creating LaTeX Workshop http and websocket server.
[02:18:06] LaTeX Workshop initialized.
[02:18:06] Found root file from active editor: /home/fernando/Downloads/test/Untitled.tex
[02:18:06] Root file changed from: undefined to /home/fernando/Downloads/test/Untitled.tex. Find all dependencies.
[02:18:06] Instantiating a new file watcher for /home/fernando/Downloads/test/Untitled.tex
[02:18:06] Creating file watcher for .bib files.
[02:18:06] Parsing /home/fernando/Downloads/test/Untitled.tex
[02:18:06] Found .bib file /home/fernando/Downloads/test/database.bib
[02:18:06] Adding .bib file /home/fernando/Downloads/test/database.bib to bib file watcher.
[02:18:06] Parsing .bib entries from /home/fernando/Downloads/test/database.bib
[02:18:06] Parse fls file.
[02:18:06] Found .bib file /home/fernando/Downloads/test/database.bib
[02:18:06] Server created on 127.0.0.1:36017
[02:18:06] Adding /home/fernando/Downloads/test/Untitled.tex to file watcher.
[02:18:06] LaTeX Workshop version: 8.9.0
[02:18:06] Snippet data loaded.
[02:18:06] Parsed 44 bib entries from /home/fernando/Downloads/test/database.bib.
[02:18:23] Bib file watcher - responding to change in /home/fernando/Downloads/test/database.bib
[02:18:23] Parsing .bib entries from /home/fernando/Downloads/test/database.bib
[02:18:23] /home/fernando/Downloads/test/database.bib changed. Auto build project.
[02:18:23] BUILD command invoked.
[02:18:23] Building root file: /home/fernando/Downloads/test/Untitled.tex
[02:18:23] Build root file /home/fernando/Downloads/test/Untitled.tex
[02:18:23] Recipe step 1: latexmk, -synctex=1,-interaction=nonstopmode,-file-line-error,-pdf,-outdir=/home/fernando/Downloads/test,/home/fernando/Downloads/test/Untitled
[02:18:23] cwd: /home/fernando/Downloads/test
[02:18:23] LaTeX build process spawned. PID: 17046.
[02:18:23] Parsed 44 bib entries from /home/fernando/Downloads/test/database.bib.
[02:18:23] File watcher: responding to change in /home/fernando/Downloads/test/Untitled.tex
[02:18:23] Parsing /home/fernando/Downloads/test/Untitled.tex
[02:18:23] Found .bib file /home/fernando/Downloads/test/database.bib
[02:18:23] Parse fls file.
[02:18:23] Auto Build Run is temporarily disabled during a second.
[02:18:24] LaTeX log parsed with 0 messages.
[02:18:24] Recipe of length 1 finished. PID: 17046.
[02:18:24] Successfully built /home/fernando/Downloads/test/Untitled.tex.
[02:18:24] No PDF viewer connected for /home/fernando/Downloads/test/Untitled.pdf
[02:18:24] Parse fls file.
[02:18:24] Found .bib file /home/fernando/Downloads/test/database.bib
[02:18:24] Found root file from active editor: /home/fernando/Downloads/test/Untitled.tex
[02:18:24] Root file remains unchanged from: /home/fernando/Downloads/test/Untitled.tex.
[02:24:20] LOG command invoked.

### Developer Tools Console [Required]

```
To access the log, click 'help' -> 'Toggle Developer Tools' -> 'Console'. Paste anything suspicious.

[Paste the log here]
```
It's empty.

## Screenshots

A screencast.

![bug](https://user-images.githubusercontent.com/17141976/82849502-9f4e2c80-9ef8-11ea-9d4e-773722a68234.gif)

## Desktop [Required]
 - OS: Ubuntu Bionic
 - VS Code version: 1.45.1
 - Extension version: 8.9.0
 - TeX distribution version: TeX Live 2017

### Are you using LaTeX Workshop with VS Code Remote?

No

If the answer is Yes, please write which one you are using. Write the versions of the remote extension.

- Remote Containers/Remote SSH/Remote WSL
- Remote Extension Version: [e.g. 0.48.0]

### Additional context

It happens with all documents I've tried.

**Comment by tamuratak** [comment]
Duplicate of #2081
